llvm/test/CodeGen/M68k/register-spills.ll

```diff
@@ -0,0 +1,499 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=m68k -O0 %s -o - | FileCheck %s
+
+define zeroext i1 @get1() {
```

Comment:
could you split these three `get` functions and the code they're testing into a separate patch? they're not directly related to the register spilling issues you're fixing here.

Reply:
They do test the changes made in this patch though, no? The assembly demonstrates movem being used to spill registers and 8-bit registers being assigned to 16-bit slots on the stack.

Reply:
I meant `get1`, `get8`, `get16`, and `get32`. I don't think they use movem.